src/instruments/src/SD/Pages/Press/Press.tsx

```diff
@@ -0,0 +1,556 @@
+                <text className={`Standard End ${deltaPsi < -0.4 || deltaPsi >= 8.5 ? 'Amber' : 'Green'}`} x={dpx + 63} y={y + 25}>{deltaPress[1]}</text>
+                <GaugeComponent x={dpx} y={y} radius={radius} startAngle={210} endAngle={50} visible className="Gauge">
+                    <GaugeComponent x={dpx} y={y} radius={radius} startAngle={40} endAngle={50} visible className="Gauge Amber" />
+                    <GaugeComponent x={dpx} y={y} radius={radius} startAngle={210} endAngle={218} visible className="Gauge Amber" />
+                    <GaugeMarkerComponent value={8} x={dpx} y={y} min={-1} max={9} radius={radius} startAngle={210} endAngle={50} className="GaugeText" showValue textNudgeY={10} />
+                    <GaugeMarkerComponent
+                        value={4}
+                        x={dpx}
+                        y={y}
+                        min={-1}
+                        max={9}
+                        radius={radius}
+                        startAngle={210}
+                        endAngle={50}
+                        className="GaugeText"
+                    />
+                    <GaugeMarkerComponent
+                        value={0}
+                        x={dpx}
+                        y={y}
+                        min={-1}
+                        max={9}
+                        radius={radius}
+                        startAngle={210}
+                        endAngle={50}
+                        className="GaugeText"
+                        showValue
+                        textNudgeY={-10}
+                        textNudgeX={5}
+                    />
+                    <GaugeMarkerComponent
+                        value={deltaPsi}
+                        x={dpx}
+                        y={y}
+                        min={-1}
+                        max={9}
+                        radius={radius}
+                        startAngle={210}
+                        endAngle={50}
+                        className={`GaugeIndicator ${deltaPsi < -0.4 || deltaPsi >= 8.5 ? 'Amber' : ''}`}
+                        indicator
+                    />
+                </GaugeComponent>
+            </g>
+
+            {/* Vertical speed gauge  */}
+            <CabinVerticalSpeedComponent vsx={275} y={y} radius={radius} />
+
+            {/* Cabin altitude gauge */}
+            <g id="CaIndicator">
+                <text className="Large Center" x={cax + 15} y="80">CAB ALT</text>
+                <text className="Medium Center Cyan" x={cax + 20} y="100">FT</text>
+                <text
+                    /* eslint-disable no-nested-ternary */
+                    className={`Huge End ${Math.round(cabinAlt / 50) * 50 >= 8800 && Math.round(cabinAlt / 50) * 50 < 9550
+                        ? 'GreenTextPulse'
+                        : Math.round(cabinAlt / 50) * 50 >= 9550 ? 'Red' : 'Green'}`}
+                    /* eslint-enable no-nested-ternary */
```

Comment:
```suggestion
                    className={`Huge End ${Math.round(cabinAlt / 50) * 50 >= 8800 && Math.round(cabinAlt / 50) * 50 < 9550
                        ? 'GreenTextPulse'
                        : (Math.round(cabinAlt / 50) * 50 >= 9550 ? 'Red' : 'Green')}`}
```
I think adding parentheses to make your intentions clear should satisfy the rule. Alternately you could factor the logic into some ts code above rather than inline in the JSX.